tests/extensions_tests/detection_tests/test_detection_vis_report.py

```diff
@@ -11,6 +11,13 @@
 from chainercv.utils import ConstantReturnModel
 from chainercv.utils import DummyDataset
 
+import importlib
+try:
+    importlib.import_module('matplotlib')
```

Comment:
When I used `import`, the flake8 test failed because imported `matplotlib` was unused.